frodo.js

```diff
@@ -9,21 +9,24 @@
  * Copyright 2016 Henrik Ingo (@henrikingo)
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!